build-release.sh

```diff
@@ -5,8 +5,11 @@ set -eEuo pipefail
 eval "$(go env)"
 
 OUT_DIR="${1-dist}"
-# To override the latest git tag as the version, pass something else as the second arg.
-VERSION=${2:-$(git describe --tags --always --dirty)}
+
+if [ -z "$VERSION" ]; then
+    # To override the latest git tag as the version, pass something else as the second arg.
+    VERSION=${2:-$(git describe --tags --always --dirty)}
+fi
```

Comment:
This seems something old, is it still required? The `VERSION` value should be always set from the `configure` job.